ui/lib/client/index.tsx

```diff
@@ -50,15 +51,18 @@ function initAxios() {
     const method = (config.method as string).toLowerCase()
 
     let errCode: string
+    let content: string
     if (err.message === 'Network Error') {
       errCode = 'error.network'
     } else {
-      errCode = response?.data?.code || 'error.api.other'
-      if (errCode === 'error.api.other') {
-        errCode = response?.data?.message || err.message
-      }
+      errCode = response?.data?.code
+    }
+    if (errCode !== ERR_CODE_OTHER && i18next.exists(errCode)) {
```

Comment:
Good find! Would you also like to change `TextWithInfo/index.tsx`, which can be simplified using `i18next.exists`?

Current code base:

```typescript
  const tooltip = t(`${transKey}_tooltip`, {
    defaultValue: '',
    fallbackLng: '_',
  })
```

Reply:
Sure!